tests/rules/test_rm_root.py

```diff
@@ -1,20 +1,14 @@
 import pytest
-from thefuck.rules.rm_root import match, get_new_command
+from thefuck.rules.rm_root import match
 from thefuck.types import Command
 
 
 def test_match():
-    assert match(Command('rm -rf /', 'add --no-preserve-root'))
+    assert match(Command('rm -rf /', ''))
 
 
 @pytest.mark.parametrize('command', [
     Command('ls', 'add --no-preserve-root'),
-    Command('rm --no-preserve-root /', 'add --no-preserve-root'),
-    Command('rm -rf /', '')])
+    Command('rm', '/usr/bin/python')])
 def test_not_match(command):
     assert not match(command)
-
-
-def test_get_new_command():
-    assert (get_new_command(Command('rm -rf /', ''))
-            == 'rm -rf / --no-preserve-root')
```

Comment:
Existing tests shouldn't be touched, as there's only one new function added to the rule, namely `side_effect`. Please include tests for it.